physionet-django/user/models.py

```diff
@@ -778,6 +778,21 @@ class CredentialApplication(models.Model):
     class Meta:
         default_permissions = ('change',)
 
+    def get_training_status(self):
```

Comment:
It is possible that there are multiple training applications associated with a user, so this doesn't seem quite right to me. By selecting the first(), don't we end up ignoring training applications?

Reply:
@tompollard what about turning the two columns Training Type and Training Status into one (Training Status). So Training Status would list every training available for each user along with its status. (e.g. CITI program: Active, Datathon training: Under review).
Would that make sense?

Reply:
Makes sense to me. It might end up leading to lots of information in the single cell within the column, but perhaps not a big deal? Might be worth checking to see whether @kepaik is okay with this (or if he has better ideas!).